library/amazoncorretto

```diff
@@ -1,16 +1,32 @@
-Maintainers: Amazon Corretto Team (@corretto),
-             James Guo (@jguo11),
-             Ziyi Luo (@ziyiluo),
-             David Alvarez (@alvdavi)
+Maintainers: Amazon Corretto Team <corretto-team@amazon.com> (@corretto),
+             James Guo <junguoj@amazon.com> (@jguo11),
+             Ziyi Luo <ziyiluo@amazon.com> (@ziyiluo),
+             David Alvarez <alvdavi@amazon.com> (@alvdavi)
 
```

Comment:
This is the source of the CI failures -- you've got an extra blank line here that shouldn't exist (if you want a "global" `GitRepo:`, etc value, they need to be part of the global block/"entry", which is what `Maintainers:` is part of).

Reply:
Thanks, I'm fixing this